src/Rebing/GraphQL/Support/UploadType.php

```diff
@@ -68,4 +63,12 @@ public function parseLiteral($valueNode, ?array $variables = NULL)
     {
         throw new Error('`Upload` cannot be hardcoded in query, be sure to conform to GraphQL multipart request specification. Instead got: ' . $valueNode->kind, [$valueNode]);
     }
+
+    public static function getInstance() {
+        static $inst = null;
+        if ($inst === null) {
```

Comment:
Why is that check here? If you set `static $inst = null` then it will always be null...

Reply:
Singleton design pattern, prevent creating multi instance of UploadType, if not will cause error `Schema must contain unique named types but contains multiple types named "Upload" (see http://webonyx.github.io/graphql-php/type-system/#type-registry)`, thats why we need to give a unique name for every uploadType and now we don't